src/processors/modulation/Flanger.cpp

```diff
@@ -0,0 +1,298 @@
+#include "Flanger.h"
+#include "../BufferHelpers.h"
+#include "../ParameterHelpers.h"
+
+namespace
+{
+constexpr float rateLow = 0.5f;
+constexpr float rateHigh = 40.0f;
+constexpr float delayMs = 0.001f;
+
+const String delayTypeTag = "delay_type";
+} // namespace
+
+Flanger::Flanger (UndoManager* um) : BaseProcessor ("Flanger",
+                                                    createParameterLayout(),
+                                                    um,
+                                                    magic_enum::enum_count<InputPort>(),
+                                                    magic_enum::enum_count<OutputPort>())
+{
+    using namespace ParameterHelpers;
+    loadParameterPointer (rateParam, vts, "rate");
```

Comment:
It seems like when there is a 5th parameter it's knob is not shown properly. I wondered if we might be changing around some of the parameters anyway so I haven't looked into resizing the module, but let me know what you think of how we can organize all 5 knobs(params)


<img width="262" alt="Screen Shot 2022-10-11 at 12 51 40 PM" src="https://user-images.githubusercontent.com/43660096/195175115-e5a4c2d1-c7f5-4317-a318-e1487a802d50.png">

Reply:
The new 5-slider layout is looking pretty good!